packages/App/src/types/types.tsx

```diff
@@ -160,4 +162,35 @@ export enum DayOfWeek {
   Sun = 0,
 }
 
-export type CategoryChart = Record<Category, number >;
+export type CategoryChart = Record<Category, number>;
+
+export type textType =
```

Comment:
Is it really needed? Seems like it isn't used

Reply:
No no) it`s really needs. I use it inside chart reselector. Actually it was your idea) I had some problems with types and you helped me in office)